buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -225,35 +656,44 @@
       "value": null,
       "isSecret": true
     },
-    "PB_VsoRepoUrl": {
-      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
-    },
     "PB_VsoAccountName": {
       "value": "dn-bot"
     },
     "PB_VsoPassword": {
       "value": null,
       "isSecret": true
     },
+    "PB_Branch": {
```

Comment:
See the "Run PipeBuild.exe" step of https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=6161 

@weshaggard actually removed the `PB_Branch` variable completely last week.